database/pgx/pgx.go

```diff
@@ -479,7 +475,7 @@ func (p *Postgres) ensureVersionTable() (err error) {
 		return nil
 	}
 
-	query = `CREATE TABLE IF NOT EXISTS ` + p.quoteIdentifier(p.config.MigrationsTable) + ` (version bigint not null primary key, dirty boolean not null)`
+	query = "CREATE TABLE IF NOT EXISTS \"" + p.config.SchemaName + "\".\"" + p.config.SchemaName + "\" (version bigint not null primary key, dirty boolean not null)"
```

Comment:
1. Use [raw strings](https://golang.org/ref/spec#String_literals) so we don't need to escape quotes.
2. The 2nd `p.config.SchemaName` should be `p.config.MigrationsTable`